includes/class-main.php

```
@@ -1220,18 +1220,25 @@ private function is_css_minified( $file_path ) {
 			return true;
 		}
 
-		$css_content = $this->filesystem->get_contents( $file_path );
-		if ( ! is_string( $css_content ) ) {
+		$f = @fopen( $file_path, 'r' );
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $f. Configured minimum length is 3. (undefined)

(ShortVariable)